docs/superpowers/specs/2026-06-07-gallery-domain-design.md

```
@@ -0,0 +1,203 @@
+- `ContentResourceType.GALLERY_ALBUM` enum 값.
+
+→ 본 작업은 `domain/gallery` 패키지 신규 추가 + `MediaService` 메서드 2개 추가로 **순수 가산적(additive)** 이다.
+
+---
+
+## 2. 데이터 모델 — `V11__create_gallery.sql`
+
+V10(departments)이 마지막 → 갤러리는 **V11**. 컬럼·인덱스 관례는 V7~V10을 따른다.
+
+### gallery_albums (`BaseEntity` 상속)
+| 컬럼 | 타입 | 비고 |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)